### PR TITLE
Show state in participant list

### PR DESCRIPTION
**Pull request by marcos-cereijo-pexip**
We display an icon near the participant that indicates the following information: 

- Admit button when the guest user is waiting.
- Audio mute state. 
- Video mute state. 
- Is presenting.

### CODE REVIEW

File: webapp/src/components/Conference/ParticipantList/ParticipantList.scss [hunk 1]
```diff
@@ -20,14 +20,40 @@
...
+      .ParticipantStatus {
+        display: flex;
+        align-items: center;
+        gap: 1em;
+
+        /* Center the icon */
+        > div {
+          display: flex;
+        }
+        > div > div > div {
```
Comment (lzgrzebski) [inline webapp/src/components/Conference/ParticipantList/ParticipantList.scss:53]: this feels really fragile

**Review by lzgrzebski — APPROVED** [review]
👍

File: webapp/src/components/Conference/ParticipantList/ParticipantList.scss [hunk 2]
```diff
@@ -20,14 +20,33 @@
...
+
+      .AdmitButton {
+        min-height: 2em;
+        padding:  0 0.8em;
+      }
+
+      .ParticipantStatus {
+        display: flex;
+        align-items: center;
+        gap: 1em;
```
Comment (lzgrzebski) [inline webapp/src/components/Conference/ParticipantList/ParticipantList.scss:47]: is it conscious decision to use `em` or would it be better to use `rem`?

Comment (marcos-cereijo-pexip) [reply webapp/src/components/Conference/ParticipantList/ParticipantList.scss:47]: I always used `em`. Is it better to use `rem` instead?

**Pull request merged**